conformance/tests/admin-network-policy-core-egress-tcp-rules.go

```diff
@@ -207,12 +208,65 @@ var AdminNetworkPolicyEgressTCP = suite.ConformanceTest{
 			// ensure egress from gryffindor is PASSED to slytherin at port 80; egressRule at index3 should take effect
 			success := kubernetes.PokeServer(t, s.ClientSet, &s.KubeConfig, "network-policy-conformance-gryffindor", "harry-potter-0", "tcp",
 				serverPod.Status.PodIP, int32(80), s.TimeoutConfig.RequestTimeout, true)
-			assert.Equal(t, true, success)
+			assert.True(t, success)
 			// harry-potter-1 is our client pod in gryffindor namespace
 			// ensure egress from gryffindor is ALLOWED to slytherin for rest of the traffic; matches no rules hence allowed
 			success = kubernetes.PokeServer(t, s.ClientSet, &s.KubeConfig, "network-policy-conformance-gryffindor", "harry-potter-1", "tcp",
 				serverPod.Status.PodIP, int32(8080), s.TimeoutConfig.RequestTimeout, true)
-			assert.Equal(t, true, success)
+			assert.True(t, success)
 		})
 	},
 }
+
+var AdminNetworkPolicyEgressNamedPort = suite.ConformanceTest{
```

Comment:
should we put this test and the ingress one under `test/admin-network-policy-extended-named-ports.go` ? This file is large as is, just wondering if we should mix them up like currently or keep them separate in new file for readability?

Reply:
Yes moved them to new files